code/scripting/api/objs/ship.cpp

```diff
@@ -56,12 +56,12 @@ ADE_INDEXER(l_ShipTextures, "number Index/string TextureFilename", "Array of shi
 {
 	object_h *oh;
 	const char* s;
-	int tdx=-1;
-	if (!ade_get_args(L, "os|o", l_ShipTextures.GetPtr(&oh), &s, l_Texture.Get(&tdx)))
-		return ade_set_error(L, "o", l_Texture.Set(-1));
+	texture_h* tdx = nullptr;
```

Comment:
Maybe another one (leads to [here](https://github.com/scp-fs2open/fs2open.github.com/pull/1881/files#diff-cf534f763ab30f11e573c3e1c9daadeeR114)?)

Reply:
Another case of setting a variable.